apps/web/.env.test

```diff
@@ -78,3 +78,7 @@ OS_PLACES_API_KEY = os_places_api_key
 # TODO: remove the requirement for these props in the web .env config
 DATABASE_URL="sqlserver://"
 PORT=0
+SERVICE_BUS_PORT="5672"
```

Comment:
do we need those in the other .envs as well?

Reply:
these are the bare minimum for running the queues locally, so that no one would need to have anything additional in their `.env` files as well

Reply:
wouldn't they go to .env.local then?

Reply:
issue resolved - no vars needed outside env.test